AutotypeURL/ATUAutotypeURL.m

```diff
@@ -51,14 +52,33 @@ - (nonnull NSString *)windowTitleForRunningApplication:(nonnull NSRunningApplica
 
 - (NSString *)_getURLFromSafari {
   NSAppleScript *script = [[NSAppleScript alloc] initWithSource:@"tell application \"Safari\" to get URL of front document"];
-  NSAppleEventDescriptor *aed = [script executeAndReturnError:NULL];
-  return aed.stringValue;
+    NSAppleEventDescriptor *aed = [script executeAndReturnError:NULL];
```

Comment:
Please apply the same code style as the project (2 spaces, no tabs)

Reply:
ugh sorry....i have habit of using tabs...fixed